src/test/java/Homework17.java

```diff
@@ -59,5 +60,15 @@ public void playListpg() throws InterruptedException{
     public WebElement getSuccessShow(){
         return driver.findElement(By.cssSelector("[class= 'success show']"));
     }
+=======
+
+
+
+
+public class Homework17 extends BaseTest{
+
+
+
+>>>>>>> Stashed changes
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210742277-6a3360e4-48bb-4faf-ac22-9672f9356f52.png)